README.md

```diff
@@ -99,6 +99,25 @@ One can adjust the values to improve the accuracy of the temperature read.
 The **getTemperature()** has implemented this algorithm, however as long
 as one does not set the SC it will use the K_TC as default.
 
+## GetRawData 
+This function allows you to get all the 32 bits raw data from the board, after the standard reading.
+Example code could be found in the examples folder.
+
+```cpp
+thermocouple.read();              
+value=thermocouple.getRawData();  // Read the raw Data value from the module
+```
+## Pull Up Resistor 
```

Comment:
Pull UP story is not bad, but not all users will use PIN12 as datapin. So you need to describe that in such a way it will work for all possible pins.